docs/components/image/controller.md

```diff
@@ -0,0 +1,16 @@
+# Image reflector and automation controllers
+
+The image-reflector-controller and image-automation-controller work together to update your git
```

Comment:
```suggestion
The image-reflector-controller and image-automation-controller work together to update your Git
```

Because it is a name, unlike `git`.

Reply:
Capitalised, in two places.